src/test/java/org/mockitousage/stubbing/StubbingConsecutiveAnswersTest.java

```diff
@@ -41,6 +41,34 @@ public void should_return_consecutive_values_for_two_nulls() throws Exception {
         assertNull(mock.simpleMethod());
     }
 
+    @Test
+    public void should_return_consecutive_values_first_var_arg_null() throws Exception {
```

Comment:
This is the current behavior, correct? E.g. your PR did not change it?

Reply:
Exactly, I've added to confirm that.